includes/gateways/stripe/includes/class-give-stripe-webhooks.php

```diff
@@ -63,8 +63,14 @@ public function listen() {
 				return;
 			}
 
-			// Get the Stripe SDK autoloader.
-			require_once GIVE_PLUGIN_DIR . 'vendor/autoload.php';
+			// Load Stripe SDK.
+			$stripe_sdk_compatibility = give_get_option( 'stripe_sdk_incompatibility', 'composer' );
+
+			if ( 'composer' === $stripe_sdk_compatibility ) {
```

Comment:
@mehul0810 I already mentioned that create a function for this in `Give_Stripe`

Reply:
Created a helper function to load Stripe SDK